config/database.yml

```diff
@@ -1,21 +1,21 @@
 default: &default
-  adapter:  postgresql
+  adapter: postgresql
```

Comment:
move back changes 
call in your branch:
`git comit checkout -f config/database.yml`

Reply:
check file on `rubocop`